LICENSE

```diff
@@ -1,31 +1,11 @@
-Copyright (c) <2017-2018>, Valassis Digital
-Copyright (c) <2016-2017>, MaxPoint Interactive
-Copyright (c) <2015-2018>, Eric Dill
-Copyright (c) <2012>, Anaconda, Inc
+Copyright 2025 Pavel Zwerschke, Daniel Elsner
```

Comment:
Can you change this to conda-mirror contributors to cover future contributions?

Reply:
[`3d485c1` (#76)](https://github.com/conda-incubator/conda-mirror/pull/76/commits/3d485c1038e7c99787241a34a734b68eb7826ce1)